src/main/webapp/map.html

```diff
@@ -0,0 +1,59 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hello Google Maps</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAIJxmuluUxbW2r2rZgOebw7B0U1rRtrvA"></script>
+    <script>
+      /** Creates a map that shows landmarks around Google. */
+      function createMap(){
```

Comment:
You could update this function to take the ID of the element that you want to turn into a map and then it would be super easy to create multiple maps on the same page!